x-pack/plugins/security_solution/public/timelines/components/timeline/pin/index.tsx

```diff
@@ -29,7 +29,7 @@ export const Pin = React.memo<Props>(
     const isTemplate = timelineType === TimelineType.template;
     return (
       <EuiButtonIcon
-        aria-label={pinned ? i18n.PINNED : i18n.UNPINNED}
+        aria-label={pinned ? i18n.PINNED : isTemplate ? i18n.DISABLE_PIN : i18n.UNPINNED}
```

Comment:
```suggestion
        aria-label={isTemplate ? i18n.DISABLE_PIN : pinned ? i18n.PINNED : i18n.UNPINNED}
```
I think this logic will match better with our logic for the attribute `isDisabled`